### ISSUE TITLE
Picture 

### ISSUE THREAD

**Issue by AfonsOMLOPedro** [joaopedro/InvoiceKeeper-android#1]
Picture is not perceptible. Wen you see a picture on document, this is not PERCEPTIBLE and you don't read the feacture. The picture is Pixelized.

**Comment by joaopedro** [comment]
you're right, the quality in the picture is poor. We will increase the quality.

**Comment by joaopedro** [comment]
This only happens when  the picture is taken from the camera and not choose from the media store.
